cacheback/base.py

```diff
@@ -379,3 +379,44 @@ def process_result(self, result, call, cache_status, sync_fetch=None):
                            was required (ie the result was a cache hit).
         """
         return result
+
+    # --------------------
+    # ASYNC HELPER METHODS
+    # --------------------
+
+    def job_refresh(klass_str, obj_args, obj_kwargs, call_args, call_kwargs):
```

Comment:
I think this needs to be a `@classmethod` if you're calling it as `Job.job_refresh`. But on the other hand, is there really a need to have this on the Job class? Couldn't this just be moved from `cacheback/tasks.py` to `cacheback/cache.py` so the call in `tasks.py` and `rq_tasks.py` would be to `cacheback.cache.refresh`? This may even remove the need for the inline import in the `refresh_cache` functions.